BannerlordTwitch/BannerlordTwitch/Api/BLTApi.cs

```diff
@@ -128,11 +132,28 @@ public void OnCommand(HttpListenerRequest req, HttpListenerResponse resp)
             {
                 json.actualParams = "cmd=" + cmdName + "&pseudo=" + pseudo + "&args=" + args;
                 json.status = BLTModule.TwitchService?.TestCommand(cmdName, pseudo, args);
+
+                if(json.status == true || json.status == false)
```

Comment:
This is wrong, as response from an action (command or reward) can be delayed, that is why I use the SendReply function.
e.g. Summoning is enqueued into the MissionBehavior tick, and only replies once that is executed (which could be same frame or when mission is unpaused.

A command invoked via API like this will not send its own response via the direct reply, it can only send that it has been successfully queued.
The specific response could come later or not at all depending on the nature of the command.